build.gradle

```diff
@@ -2,73 +2,61 @@ plugins {
     id 'java'
     id 'maven-publish'
     id 'jacoco'
-    id "io.freefair.lombok" version "5.3.0"
-    id "com.github.hierynomus.license" version "0.15.0"
+    id 'io.freefair.lombok' version '5.3.0'
+    id 'com.diffplug.spotless' version '5.8.2'
 }
 
 jacoco {
     // Use JaCoCo 0.8.6 for (experimental) support for Java 15 class files.
-    toolVersion = "0.8.6"
+    toolVersion = '0.8.6'
 }
 
 group = 'com.dumbdogdiner'
-
 version = '2.0.0'
 
-// License Plugin Options
-license {
-    header = project.file('LICENSE_HEADER')
-    ext.year = Calendar.getInstance().get(Calendar.YEAR)
-    mapping("java", "SLASHSTAR_STYLE")
-}
-
 tasks.withType(JavaCompile) {
     options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation" << "-XDignore.symbol.file"
 }
-// Run the license formatter before compiling the source code.
-tasks.compileJava.dependsOn licenseFormatMain, licenseFormatTest
 
 configurations {
     jaxDoclet
     // give test dependencies access to compileOnly dependencies to emulate providedCompile
     testImplementation.extendsFrom compileOnly
 }
 
-repositories {
-    mavenCentral()
-    jcenter()
+allprojects {
+    repositories {
+        mavenCentral()
+        jcenter()
 
-    maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
-    maven { url 'https://papermc.io/repo/repository/maven-public/' }
+        maven { url 'https://oss.sonatype.org/content/repositories/snapshots' }
+        maven { url 'https://papermc.io/repo/repository/maven-public/' }
+    }
 }
 
-dependencies {
-    compileOnly 'org.projectlombok:lombok:1.18.16'
-    annotationProcessor 'org.projectlombok:lombok:1.18.16'
+subprojects {
+    apply plugin: 'io.freefair.lombok'
+    lombok {
+        config['lombok.addLombokGeneratedAnnotation'] = 'true'
+    }
 
-    compileOnly 'com.destroystokyo.paper:paper-api:1.16.4-R0.1-SNAPSHOT'
-    compileOnly 'net.md-5:bungeecord-api:1.16-R0.4-SNAPSHOT'
-    implementation 'org.jetbrains:annotations:20.1.0'
-    implementation 'com.google.code.gson:gson:2.8.6'
-    implementation 'io.github.classgraph:classgraph:4.8.92'
-    implementation 'com.github.seancfoley:ipaddress:5.3.3'
-    
-    // JUnit 5 Testing
-	testImplementation("org.junit.jupiter:junit-jupiter-api:5.7.0")
-    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.7.0")
+    apply plugin: 'com.diffplug.spotless'
+    spotless {
+        ratchetFrom 'origin/master'
 
-    // Mocking
-    testImplementation("org.powermock:powermock-module-junit4:2.0.9")
-    testImplementation("org.powermock:powermock-api-mockito2:2.0.9")
-    testImplementation("org.powermock:powermock-module-junit4-rule:2.0.9")
-    testImplementation("org.powermock:powermock-classloading-xstream:2.0.9")
-    testImplementation("org.mockito:mockito-core:3.6.28")
+        java {
+            importOrder()
+            prettier(['prettier': '2.0.5', 'prettier-plugin-java': '0.8.0']).config(['parser': 'java', 'tabWidth': 4])
```

Comment:
Isn't the default printWidth like.. 60? Can we please bump it to like.. 100/120? Of course, open to discussions

Reply:
+1, would be better at 100